web/book/src/internals/README.md

```diff
@@ -3,10 +3,22 @@
 This chapter explains PRQL's semantics: how expressions are interpreted and
 their meaning. It's intended for advanced users and compiler contributors.
 
-- [Name resolving](./name-resolving.md)
-- [Functions](./functional-lang.md)
-- [Syntax highlighting](./syntax-highlighting.md)
-
 It's also worth checking out the
 [`prql-compiler` docs](https://docs.rs/prql-compiler/latest/prql_compiler/) for
 more details on its API.
+
+## Terminology
+
+[_Relation_](<https://en.wikipedia.org/wiki/Relation_(database)>): Standard
+definition of a relation in context of databases:
+
+- An ordered set of tuples of form `(d_0, d_1, d_2, ...)`.
+- Set of all `d_x` is called an attribute or a column. It has a name and a type
+  domain `D_x`.
+
+_Frame_: descriptor of a relation. Contains list of columns (with names and
+types). Does not contain data.
```

Comment:
```suggestion
```

Reply:
This it no longer named frame, but lineage. And is in the process of being merged with `Ty`, so it can be removed from here.